src/Pages/Page.php

```diff
@@ -133,7 +133,16 @@ public function flatReferenceGroup(): string
 
     public function mediaUrls($type = null): Collection
     {
-        return $this->getAllFiles($type)->map->getFileUrl();
+        // TODO getallfiles should actually get all files...
+        // What was the creator of the assetlibrary package thinking. It sure wasn't me... I promise...
+        $assets = $this->getAllFiles($type, app()->getLocale())->map->getFileUrl();
```

Comment:
hier de default locale veronderstellen lijkt me niet juist. ik weet niet goed de impact hiervan maar inderdaad bedoeling dat alle media wordt opgehaald.

Reply:
Ik had hier default locale genomen omdat mediaUrls() enkel gebruikt word voor op te halen in de view (dacht ik).